cli/template/extras/src/env/with-auth-db.js

```diff
@@ -28,7 +28,7 @@ export const env = createEnv({
       // VERCEL_URL doesn't include `https` so it cant be validated as a URL
       process.env.VERCEL ? z.string() : z.string().url()
     ),
-    // Add ` on ID and SECRET if you want to make sure they're not empty
+    // Add `.min(1)` on ID and SECRET if you want to make sure they're not empty
```

Comment:
This line isn't needed at all with the option below set to true

Reply:
Nah, u dont need to repair the 'string' one 💀💀💀